airbyte-cdk/java/airbyte-cdk/db-sources/src/testFixtures/java/io/airbyte/cdk/testutils/NonContainer.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.testutils;
+
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+/**
+ * The class is used to make sources that don't use test containers (such as Snowflake) compatible
+ * with {@link io.airbyte.cdk.integrations.source.jdbc.test.JdbcSourceAcceptanceTest}
+ */
+public class NonContainer extends JdbcDatabaseContainer<NonContainer> {
```

Comment:
FakeContainer? Any better ideas for name @postamar ? 

Can we change the comment to something like "This is used when a source relies on a always-on resource and therefore doesn't need an actual container." If someone is interested in looking at use cases, they can just look for users of this class, that way the comment never gets stale